gitreceived/gitreceived.go

```diff
@@ -201,6 +196,21 @@ func handleChannel(conn *ssh.ServerConn, newChan ssh.NewChannel) {
 				ch.Stderr().Write([]byte("Invalid repo.\n"))
 				return
 			}
+
+			cacheKey := cmdargs[1]
+			if *cacheKeyHook != "" {
+				var result bytes.Buffer
+				var errout bytes.Buffer
+				cmd := exec.Command(*cacheKeyHook, cmdargs[0], cmdargs[1])
+				cmd.Stdout = &result
+				cmd.Stderr = &errout
```

Comment:
Merged my changes incorrectly, it's used properly now. Stderr from the script gets turned into an error after `cmd.Run()`, so we can see what actually went wrong.